src/pages/Admin/Charity/Dashboard/Balance.tsx

```diff
@@ -0,0 +1,46 @@
+import { PropsWithChildren } from "react";
+import { humanize } from "helpers";
+
+type Props = {
+  type: "Sustainability Fund" | "Donations";
+  current: number;
+  pending: number;
+  payedOut: number;
+};
+
+export default function Balance({ type, current, pending, payedOut }: Props) {
+  return (
+    <div className="@container rounded border border-prim bg-orange-l6 dark:bg-blue-d6 p-4">
+      <h4 className="uppercase text-sm @xs:text-xl font-bold mb-5">
+        {type} account
+      </h4>
+      <p className="text-xs @xs:text-sm text-gray-d1 dark:text-gray mb-8">
+        Note: balances are estimations.
+      </p>
+      <div className="grid @xs:grid-cols-[auto_1fr] gap-y-5 justify-self-start gap-x-2 @xs:gap-x-8">
+        <Amount title="Current balance" classes="col-span-full @xs:mr-auto">
```

Comment:
Current Balance === Pending Current Payouts. In other words, there's really no current balance. There is however the idea of total current donations received, which would be a matter of changing the label here. 
```suggestion
        <Amount title="Total Current Contributions" classes="col-span-full @xs:mr-auto">
```

Reply:
See https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2480/commits/585eb2aa98491660d2acdc067f2bd5bdf581560e